frontend/src/metabase/questions/actions.ts

```diff
@@ -11,22 +11,56 @@ export interface LoadMetadataOptions {
 }
 
 export const loadMetadataForCard =
-  (card: Card, options?: LoadMetadataOptions) =>
+  (card: Card, options?: LoadMetadataOptions) => async (dispatch: Dispatch) => {
+    await dispatch(loadDependentMetadata(card, [], options));
+  };
+
+const loadDependentMetadata =
+  (
+    card: Card,
+    dependencies: Lib.DependentItem[],
+    options?: LoadMetadataOptions,
+  ) =>
   async (dispatch: Dispatch, getState: GetState) => {
     const question = new Question(card, getMetadata(getState()));
```

Comment:
As before, we need to re-create the question instance after each metadata change